src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,36 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  if (!comments) {
+    return null;
+  }
```

Comment:
Not necessary for the tests, but can be considered a good practice in some instances.

Reply:
sometimes yes, sometimes it's good practice.
But in ur case, in addition to comments, you also render information about the user, which will not be displayed.
as I see - u already have a check for empty comments below - `{comments.length > 0 ? (`